pkg/authentication/basic/htpasswd.go

```diff
@@ -139,6 +139,16 @@ func passShaOrBcrypt(h *htpasswdMap, user, password string) (invalidEntries []st
 	return invalidEntries
 }
 
+func (h *htpasswdMap) GetUsers() map[string]interface{} {
```

Comment:
Exported functions should have a comment, please make sure to add a suitable comment to the function